### ISSUE TITLE
Accessible view doesn't properly respect the bounds of the VS Code window

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#188838]
Testing #188537

<img width="992" alt="image" src="https://github.com/microsoft/vscode/assets/4544166/09bce88e-52a9-49f6-be62-72ba81498360">

Here's the accessible view consuming my entire VS code window.

This just looks really bad, I wonder if it should be scrollable when the content is sufficiently large

**Comment by meganrogge** [comment]
Yeah wasn't really a priority since this is a feature for screen reader users, but we can add this. @Amunger mentioned this earlier.

**Comment by lramos15** [comment]
Yeah that was my initial thought, but the amount of vision is a spectrum and screenreader users may still be able to identify some elements on the screen and this obscures even the larger identifiable elements like the activity bar
